man/man1/p11sak.1.in

```diff
@@ -1,678 +1,1759 @@
 .TH P11SAK 1 "May 2020" "@PACKAGE_VERSION@" "openCryptoki"
 .SH NAME
-p11sak \- generate and list token keys in an openCryptoki token repository.
+p11sak \- Manage token keys in a PKCS\~#11 token repository.
+.
 .
 .
 .SH SYNOPSIS
 .B p11sak
 .I command
-.RI [ ARGS ]
+.RB [ ARGUMENTS ]
 .RB [ OPTIONS ]
-.
 .PP
+.
 .B p11sak
 .BR \-\-help | \-h
 .br
-
-.SH DESCRIPTION
-.B p11sak can be used to generate, list and delete the token keys in an openCryptoki token repository.
-The utility provides a flexible key management tool in openCryptoki to list and generate symmetric (DES, 3DES, AES, AES-XTS) and asymmetric (RSA, EC, IBM Dilithium, IBM Kyber) keys.
-This tool is especially capable of a well defined listing of keys with their PKCS #11 attributes.
-.
+.B p11sak
+.BR \-\-version | \-v
 .
 .
-.SH COMMANDS
-The \fBp11sak\fP tool can operate in three modes: when command
-.I generate-key
-is specified, it operates in the mode to generate a token key in the openCryptoki token repository.
-If command
-.I list-key
-is given, it lists the keys specified in the arguments.
-If command
-.I remove-key
-is given, it removes the keys specified in the arguments.
 .
+.SH DESCRIPTION
+The \fBp11sak\fP tool can be used to manage token keys in a PKCS\~#11 token
+repository. The utility provides a flexible key management tool to generate,
+list, remove, update, copy, import, and export symmetric (DES, 3DES, generic,
```

Comment:
delete ?

Reply:
Since we stay with remove, we keep remove here as well.